### DISCUSSION TITLE
Using Style Dictionary with MantineProvider theme and or overriding styles

### DISCUSSION THREAD

**Discussion by diegodesouza** [mantinedev/mantine#6248] [Q&A]
I'm wondering if it is possible to use Style Dictionary along with MantineProvider? I would like the option to create tokens and have Style Dictionary build the scss variables, and then use them to override some component styling to match my designs. But also have the ability of passing themes to the MantineProvider. But i have a feeling that the theme provider would override my tokens? or conflict with it? have you see this type of config or use?

**Comment by rtivital** [comment]
MantineProvider is a runtime js component, it cannot transform anything that is defined at build time.

**Comment by diegodesouza** [comment]
makes sense @rtivital thanks for the prompt response.
